lib/immortal.rb

```diff
@@ -1,37 +1,38 @@
 require 'immortal/belongs_to'
 
 module Immortal
-
   def self.included(base)
     base.send :extend, ClassMethods
     base.send :include, InstanceMethods
     base.send :include, BelongsTo
     base.class_eval do
       class << self
-        alias :mortal_delete_all :delete_all
-        alias :delete_all :immortal_delete_all
+        alias_method :mortal_delete_all, :delete_all
+        alias_method :delete_all, :immortal_delete_all
       end
     end
   end
 
   module ClassMethods
-
+    # @return [Boolean] whether the model supports immortal or not
     def immortal?
-      self.included_modules.include?(::Immortal::InstanceMethods)
+      included_modules.include?(::Immortal::InstanceMethods)
     end
 
     def without_default_scope
-      new_scope = self.unscoped
-      our_scope = self.current_scope || self.unscoped
+      new_scope = unscoped
+      our_scope = current_scope || unscoped
 
       non_immortal_constraints_sql = our_scope.arel.constraints.to_a.map do |constraint|
-        constraint.to_sql.split('AND').reject{|clause| clause.include?('deleted')}
+        constraint.to_sql.split('AND').reject { |clause| clause.include?('deleted') }
       end.flatten.join(' AND ')
 
       new_scope = new_scope.merge(our_scope.except(:where))
       new_scope = new_scope.where(non_immortal_constraints_sql)
 
-      unscoped.merge(new_scope).scoping do
+      unscoped
+        .merge(new_scope)
+        .scoping do
```

Comment:
Why this change? It seems a little bit harder to read this multiple line chain and then the `do`.